packages/venia-concept/src/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -922,12 +952,18 @@ describe('toggleCart', () => {
 
     test('its thunk opens the drawer and refreshes the cart', async () => {
         await toggleCart()(...thunkArgs);
+        // toggleDrawer
+        expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
+        expect(dispatch).toHaveBeenCalledTimes(1);
 
+        getState.mockReturnValueOnce({
+            app: { drawer: 'cart' },
+            cart: { details: { items_count: 2 } }
+        });
+        await toggleCart()(...thunkArgs);
+        // Refresh the cart
+        expect(dispatch).toHaveBeenNthCalledWith(2, expect.any(Function));
```

Comment:
And because this setup and assertion duplicates your previous test "its thunk opens the drawer when the cart has items", you can drop this too.

Therefore you can drop this entire test 🎉

Reply:
I've tried to fixed it.please review and let me know.